app/models/post.rb

```diff
@@ -0,0 +1,14 @@
+class Post < ApplicationRecord
+  belongs_to :user
+  has_many :comments
+  has_many :likes
+
+  def increment_user_posts_count
+    user.posts_counter += 1
+    user.save
+  end
```

Comment:
- [ ] Your method is good. However there are more clean and efficient way to it. 
 You can put the increment method after save. Like that:

```
  after_save :update_post_counter

  private

  def update_post_counter
    author.increment!(:posts_counter)
  end
```

Reply:
This change has been done well good work